Stratis.Bitcoin.Api/ApiFeature.cs

```diff
@@ -51,6 +53,14 @@ public override void Stop()
         {
             if (this.asyncLoop != null)
                 this.asyncLoop.Dispose();
+
+            // Make sure we are releasing the listening ip address / port
+            if (this.webHost != null)
+            {
+                this.logger.LogInformation($"Api stopping on url {this.fullNode.Settings.ApiUri}");
+                this.webHost.StopAsync(TimeSpan.FromSeconds(10)).Wait(TimeSpan.FromSeconds(15));
```

Comment:
I would make the 10 seconds a constant at the top of the class.
Why use `.Wait(TimeSpan)` here? 
In any case the StopAsync will terminate within 10 seconds, won't it?

Reply:
Changed.